src/main/java/gov/usgs/locator/LocService.java

```diff
@@ -99,7 +103,7 @@ public LocOutput getLocation(final LocInput in) throws LocationException {
     // init the tt models
     TTSessionLocal ttLocal = null;
     try {
-      ttLocal = new TTSessionLocal(true, true, true, modelPath);
+      ttLocal = new TTSessionLocal(true, true, true, modelPath, serializedPath);
     } catch (IOException | ClassNotFoundException e) {
```

Comment:
I tried to find where this exception mapped back to the locatorExitCode for bad aux data, but my searches did not find it. So just making sure this LocationException with a free text string maps to the LocatorExitCode which is translated to LocatorStatus for return the caller, right? I was just a little surprised there was a free text string in the exception when there are also well defined constants, so Thanks for confirming.

Reply:
I just mapped ray's codes to the output format, I assume his code for bad aux data is generated by something more subtle than "file not found"